UPGRADE-1.13.md

```diff
@@ -367,3 +371,5 @@ List of affected classes:
    - `MessageDispatcher` will be migrated  to `CommandDispatcher`.
    - `MessageHandler` will be migrated  to `CommandHandler`. 
    - Example: Within the `Sylius\Bundle\CoreBundle`, the `MessageHandler\OrderHandler` class will be migrated to `CommandHandler\OrderHandler`. This pattern will be mirrored across other bundles in the system.
+
+1. The behavior of the `sylius:install:setup` command has changed, because `Sylius\Bundle\CoreBundle\Installer\Setup\LocaleSetup` has been updated. Now, it automatically replaces the existing `locale` parameter in the configuration with the one provided for the store.
```

Comment:
```suggestion
1. The behavior of the `sylius:install:setup` command has changed, because `Sylius\Bundle\CoreBundle\Installer\Setup\LocaleSetup` has been updated.
   Now, it automatically replaces the existing `locale` parameter in the configuration with the one provided for the store.
```